Microsoft.Toolkit.Uwp.PlatformSpecificAnalyzer/Microsoft.Toolkit.Uwp.PlatformSpecificAnalyzer.csproj

```diff
@@ -6,22 +6,6 @@
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
   </PropertyGroup>
   
-  <PropertyGroup>
```

Comment:
You should still have <Title> <Description> and <PackageTags> here (see other projects for example)